app/grandchallenge/components/form_fields.py

```diff
@@ -26,6 +32,9 @@ def _join_with_br(a, b):
         return b
 
 
+INTERFACE_FORM_FIELD_PREFIX = "interface_field_"
```

Comment:
Would it also work if we threw in an underscore or two? Just a bit of extra safety:

```suggestion
INTERFACE_FORM_FIELD_PREFIX = "__INTERFACE_FIELD__"
```